test/input/geo/PoconoSampleData2/PoconoRouteLink.shp.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file and .shx were missing, causing the shapefile to be unviewable with GIS (it was just fine for use with the application -- fiona wasn't reading the index, apparently.)